librz/core/csign.c

```diff
@@ -95,15 +95,24 @@ static void flirt_print_node(const RzFlirtNode *node, int indent) {
 RZ_API void rz_core_flirt_dump(RZ_NONNULL const char *flirt_file) {
 	rz_return_if_fail(RZ_STR_ISNOTEMPTY(flirt_file));
 
+	const char *extension = rz_str_lchr(flirt_file, '.');
+	if (RZ_STR_ISEMPTY(extension) || (strcmp(extension, ".sig") != 0 && strcmp(extension, ".pac") != 0)) {
```

Comment:
I think it's better to determine by the file magic (if there is any) than by extension.

Reply:
pac does not have any magic

Reply:
also for compressed flirts, there is a magic check